### PR TITLE
Install CNV operator first

### PR DESCRIPTION
**Pull request by ebattat**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [x] enhancement
- [ ] documentation
- [ ] dependencies

### CODE REVIEW

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]
Please add a comment explaining why they must be done in this order, so no one in the future changes it without understanding the implications.

**Comment by ebattat** [comment]
Right, added !!

**Comment by ebattat** [comment]
@jeniferh, what we need to change in [dv yaml ](https://github.com/redhat-performance/benchmark-runner/blob/main/benchmark_runner/common/template_operations/templates/windows/internal_data/windows_dv_template.yaml)?

**Review by RobertKrawitz — APPROVED** [review]

**Review by jeniferh — APPROVED** [review]
Looks good. When installed in this order (for 4.14+) you should see this storage class automatically created:
`ocs-storagecluster-ceph-rbd-virtualization`
In the Win bootstorm DV just replace w/ that new name:
`storageClassName:  ocs-storagecluster-ceph-rbd-virtualization`

File: benchmark_runner/common/template_operations/templates/windows/internal_data/windows_dv_template.yaml [hunk 1]
```diff
@@ -16,4 +16,4 @@ spec:
       requests:
         storage: {{ storage }}
     volumeMode: Block
-    storageClassName: ocs-storagecluster-ceph-rbd
+    storageClassName: ocs-storagecluster-ceph-rbd-virtualization
```
Comment (ebattat) [inline benchmark_runner/common/template_operations/templates/windows/internal_data/windows_dv_template.yaml:19]: @jeniferh, just here right ?

Comment (jeniferh) [reply benchmark_runner/common/template_operations/templates/windows/internal_data/windows_dv_template.yaml]: Yeah

**Pull request merged**